system/version.py

```diff
@@ -69,12 +69,17 @@ class OpenpilotMetadata:
   git_commit: str
   git_origin: str
   git_commit_date: str
+  build_style: str
   is_dirty: bool  # whether there are local changes
 
   @property
   def short_version(self) -> str:
     return self.version.split('-')[0]
 
+  @property
+  def short_commit(self) -> str:
+    return self.git_commit[:6]
```

Comment:
6 characters might not be enough to unambiguously identify commits; `git log --abbrev-commit` uses 9 chars in openpilot

Reply:
well it should be a unique identifier so lets do 9